src/routes/search.js

```diff
@@ -104,6 +109,10 @@ class SearchRoute extends Component {
     this.setState({ sort, searchFilter }, () => this.handleSearch());
   }
 
+  handleTagFilterChange = (searchFilter) => {
+    this.setState({ searchFilter, searchString: '' }, () => this.handleSearch());
```

Comment:
`() => this.handleSearch()` can be replaced with `this.handleSearch`

Reply:
Thanks Updated!